### ISSUE TITLE
GHA - unable to add extra: --namespace -unknown flag

### ISSUE THREAD

**Issue by abacchilb** [Legit-Labs/legitify#298]
### TL;DR

Using the github action- when I want to just scope the analysis to certain namespaces and I pass the namespace flag to extras- I get errors.

### Expected behavior

extra: "-n organization,actions"

Tool scans for organization and actions policies. 

### Observed behavior

for: extra: "--namespace organization"

Error: The process '/home/runner/work/_actions/Legit-Labs/legitify/main/legitify' failed with exit code 1 | stderr: Error: unknown flag: --namespace organization
2024/04/11 01:11:34 2024/04/11 01:11:34 error executing command: unknown flag: --namespace organization

for: extra: "-n actions"

Error: The process '/home/runner/work/_actions/Legit-Labs/legitify/main/legitify' failed with exit code 1 | stderr: 

---> If you have any questions or you need assistance using legitify, please don't hesitate REACHING OUT @ legitify@legitsecurity.com
Error: invalid namespace  actions

### Version

1.0.7

### On which operating system are you using legitify?

Linux

### Additional information

I can run `legitify analyze --namespace organization,actions` in my cli and it works great with the same PAT.

Example GHA- 

```

jobs:
  analyze_all:
    runs-on: ubuntu-latest
    steps:
      - name: Legitify Action
        uses: Legit-Labs/legitify@main
        with:
          github_token: ${{ secrets.PAT_FOR_LEGITIFY }}
          extra: "-n actions"
```

**Comment by noamd-legit** [comment]
Thanks for opening the issue @abacchilb.
A fix will be available soon:
https://github.com/Legit-Labs/legitify/pull/300

**Comment by noamd-legit** [comment]
merged.
